SML/SML/Matrix/Matrix.cs

```diff
@@ -0,0 +1,312 @@
+using System.Text;
+using SML.Matrices.Exceptions;
+
+namespace SML.Matrices;
+
+public class Matrix : ICloneable
+{
+    #region Fields
+
+    public int Rows { get; }
+
+    public int Columns { get; }
```

Comment:
Can you write like this?
```suggestion
    public int Rows, Columns  { get; }
```